simulariumio/data_objects/simularium_file_data.py

```diff
@@ -0,0 +1,42 @@
+from typing import Dict, Union
+from .trajectory_data import TrajectoryData
+from .frame_data import FrameData
+
+
+class SimulariumFileData:
+    def __init__(self, file_contents: Union[str, bytes]):
+        pass
+
+    def get_frame_at_index(self, frame_number: int) -> Union[FrameData, None]:
+        """
+        Return frame data for frame at index. If there is no frame at the index,
+        return None.
+        """
```

Comment:
Not sure why I didn't notice this before but are you supposed to put `pass` in each of these functions?  Also it looks like this is supposed/intended tobe an abstract base class 
```
from abc import ABC

class MyABC(ABC):
...
@abstractmethod...
```
Is that sort of the intention here?

Reply:
autoflake doesn't like it when you have `pass` after a docstring, so maybe I'll move the docstrings to the subclasses? I hadn't even noticed that `pass` had been removed, it just disappeared when I ran autoflake locally. Good eye